analyzers/AliceTestGun.py

```diff
@@ -0,0 +1,74 @@
+from heppy.framework.analyzer import Analyzer
```

Comment:
remove file ? 
don't we have a gun already? 
why a specific one? 
in any case use generic and explicit names